### DISCUSSION TITLE
disable self-CC sell/transfer

### DISCUSSION THREAD

**Discussion by mael-bomane** [deso-protocol/dips#243] [Large CIPs]
**TL;DR : %FR is your only legit cash-flow, you are Content Market Maker**

When I discovered Deso, I faithfully bought some random Creator Coin (CC), got rugged and left the platform for several months.
This alone explains a lot about low user retention, at least from my own experience and I know I'm not an isolated case.

If nothing is made to solve these repetitive scam schemes new users are facing, the team may be held accountable for allowing these, and a collective action may be taken in front of the Security and Exchange Comission (SEC) of the United States of America against the Deso Foundation, for deliberately acting against their Users, Costumers and Stakeholders best interests.

Here's a draft of what I'm proposing : https://github.com/EsyWin/dip/blob/main/dip.md 

I'm open to suggestions, improvements and comments : that's collaborative work :)
